src/PostalCodes.UnitTests/PostalCodeFormatTests.cs

```diff
@@ -0,0 +1,124 @@
+﻿using System;
+using NUnit.Framework;
+using System.Text.RegularExpressions;
+using System.Xml.Linq;
+using System.Xml;
+
+namespace PostalCodes.UnitTests
+{
+    [TestFixture]
+    public class PostalCodeFormatTests
+    {
+        [Test]
+        public void Equals_WithEqualObjects_ReturnsTrue()
+        {
+            var o1 = new PostalCodeFormat {
+                Name = "5-Digits - 99999",
+                RegexDefault = new Regex("^[0-9]{5}$", RegexOptions.Compiled),
+                OutputDefault = "xxxxx",
+                RegexShort = new Regex("aa", RegexOptions.Compiled),
+                OutputShort = "asdasd",
+                AutoConvertToShort = false,
+                ShortExpansionAsLowestInRange = "0",
+                ShortExpansionAsHighestInRange = "9",
+                LeftPaddingCharacter = "0",
+            };
+
+            var o2 = new PostalCodeFormat {
+                Name = "5-Digits - 99999",
+                RegexDefault = new Regex("^[0-9]{5}$", RegexOptions.Compiled),
+                OutputDefault = "xxxxx",
+                RegexShort = new Regex("aa", RegexOptions.Compiled),
+                OutputShort = "asdasd",
+                AutoConvertToShort = false,
+                ShortExpansionAsLowestInRange = "0",
+                ShortExpansionAsHighestInRange = "9",
+                LeftPaddingCharacter = "0",
+            };
+
+            Assert.IsTrue(o1.Equals(o2));
+            Assert.IsTrue(o2.Equals(o1));
+            Assert.IsTrue(o1.Equals(o1));
+            Assert.IsTrue(o2.Equals(o2));
+        }
+
+        [Test]
+        public void Equals_WithInequalObjects_ReturnsFalse()
+        {
+            var o1 = new PostalCodeFormat {
+                Name = "5-Digits - 99999",
+                RegexDefault = new Regex("^[0-9]{5}$", RegexOptions.Compiled),
+                OutputDefault = "xxxxx",
+                RegexShort = new Regex("aaaaaaaaa", RegexOptions.Compiled),
+                OutputShort = "asdasd",
+                AutoConvertToShort = false,
+                ShortExpansionAsLowestInRange = "0",
+                ShortExpansionAsHighestInRange = "9",
+                LeftPaddingCharacter = "0",
+            };
+           
+            var x = new PostalCodeFormat {
+                Name = o1.Name,
+                RegexDefault = o1.RegexDefault,
+                OutputDefault = o1.OutputDefault,
+                RegexShort = o1.RegexShort,
+                OutputShort = o1.OutputShort,
+                AutoConvertToShort = o1.AutoConvertToShort,
+                ShortExpansionAsLowestInRange = o1.ShortExpansionAsLowestInRange,
+                ShortExpansionAsHighestInRange = o1.ShortExpansionAsHighestInRange,
+                LeftPaddingCharacter = o1.LeftPaddingCharacter,
+            };
```

Comment:
The previous "Equals" test case would be clearer this way too, using `o1.foo` instead of copy paste.